site/_data/project.yml

```diff
@@ -48,10 +47,31 @@ commits_list_archive_markmail:
 
 jira: BAHIR
 
-source_repository: https://git-wip-us.apache.org/repos/asf/bahir.git
-source_repository_mirror: https://github.com/apache/bahir
-
 website_repository: https://git-wip-us.apache.org/repos/asf/bahir-website.git
 website_repository_mirror: https://github.com/apache/bahir-website
 
-podling: false
\ No newline at end of file
+podling: false
+
+# settings specific to supported platforms
+
+# Apache Spark
+
+spark_unix_name: bahir
+spark_github_project_name: bahir
+
+spark_latest_release: 2.1.0
+spark_latest_release_date: 02/22/2017
+
+spark_source_repository: https://git-wip-us.apache.org/repos/asf/bahir.git
+spark_source_repository_mirror: https://github.com/apache/bahir
+
+# Apache Flink
+
+flink_unix_name: bahir-flink
+flink_github_project_name: bahir-flink
+
+flink_latest_release: TBA
```

Comment:
leave blank (null) to not render dependent content, like release notes, download table, download big finger button, etc